even_fib_nums.js

```diff
@@ -7,20 +7,38 @@
  */
 function _sumFibs( maxFibValue ) {
   let sum = 0;
-
-  // do your work here
+  
+  // fibonacci
+  let testArr = [0, 1];
```

Comment:
Instead of using an array, try to think about how you can use 3 variables to keep track of the current fib number, and the two previous fib numbers. Then you can do the summation on the fly rather than traversing through a second array.